### ISSUE TITLE
Fatal Exception: NSInternalInconsistencyException

### ISSUE THREAD

**Issue by alexandreraulin** [Netvent/storyly-mobile#122]
Hi, I have a major crash with the iOS SDK. 
In view of the logs, the SDK try to update the collectionView outside of the main thread

## Environment Details
**Storyly SDK Version:** 1.9.5
**Device Model:** All
**Device OS Version:** mainly iOS 14 due to my user devices base but all iOS versions seems to be affected
**Additional Info:** /

## Expected Behavior
No crash

## Current Behavior
The SDK crash 

```
Fatal Exception: NSInternalInconsistencyException
Modifications to the layout engine must not be performed from a background thread after it has been accessed from the main thread.
```

```
Fatal Exception: NSInternalInconsistencyException
0  CoreFoundation                 0x197da0298 __exceptionPreprocess
1  libobjc.A.dylib                0x1abafa480 objc_exception_throw
2  CoreAutoLayout                 0x1abd5bc5c -[NSISEngine tryToOptimizeReturningMutuallyExclusiveConstraints]
3  CoreAutoLayout                 0x1abd5b934 -[NSISEngine _optimizeWithoutRebuilding]
4  CoreAutoLayout                 0x1abd5b494 -[NSISEngine optimize]
5  CoreAutoLayout                 0x1abd57188 -[NSISEngine performPendingChangeNotifications]
6  UIKitCore                      0x19ab306ec -[UIView(Hierarchy) layoutBelowIfNeeded]
7  UIKitCore                      0x199de2064 -[UICollectionView _performBatchUpdates:completion:invalidationContext:tentativelyForReordering:animator:animationHandler:]
8  Storyly                        0x108a65f50 _hidden#9691_ + 29 (__hidden#9976_:29)
9  Storyly                        0x108a65af8 _hidden#5486_ + 41 (__hidden#9775_:41)
10 Storyly                        0x108a21894 _hidden#5287_ (__hidden#921_)
11 Storyly                        0x108a8d1a0 _hidden#12406_ + 71 (__hidden#12629_:71)
12 Storyly                        0x108a8ee78 _hidden#12410_ + 137 (__hidden#12629_:137)
13 Storyly                        0x108a8d134 _hidden#12405_ + 69 (__hidden#12629_:69)
14 Storyly                        0x108a8d618 _hidden#12407_ + 78 (__hidden#12629_:78)
15 Storyly                        0x108a90cf8 _hidden#12416_ + 225 (__hidden#12629_:225)
16 Storyly                        0x108a93d88 _hidden#12463_
17 Storyly                        0x108a8a840 _hidden#2769_ (__hidden#921_)
18 CFNetwork                      0x19839bac4 CFNetServiceBrowserSearchForServices
19 CFNetwork                      0x1983ac584 _CFHTTPMessageSetResponseProxyURL
20 libdispatch.dylib              0x1979d72b0 _dispatch_call_block_and_release
21 libdispatch.dylib              0x1979d8298 _dispatch_client_callout
22 libdispatch.dylib              0x1979b4344 _dispatch_lane_serial_drain$VARIANT$armv81
23 libdispatch.dylib              0x1979b4e60 _dispatch_lane_invoke$VARIANT$armv81
24 libdispatch.dylib              0x1979be66c _dispatch_workloop_worker_thread
25 libsystem_pthread.dylib        0x1e01ed5bc _pthread_wqthread
26 libsystem_pthread.dylib        0x1e01f086c start_wqthread
```

## Steps to Reproduce
Maybe : see all stories from an account, add new stories from the admin web page and go back to the app with storyly view

**Comment by levent-oral** [comment]
Hello @alexandreraulin ,

We've made a fix for this issue in [iOS SDK 1.10.1](https://github.com/Netvent/storyly-mobile/blob/master/ios/RELEASENOTES.md#1101). Please check with latest version and let us know if the issue persists.

Thanks!

**Comment by alexandreraulin** [comment]
Hi @levent-oral , thanks, I will test this

**Comment by alexandreraulin** [comment]
Hi @levent-oral, the issue is no longer present in the 1.10.1 version. Thanks 👍

**Comment by levent-oral** [comment]
Hello @alexandreraulin,

Thanks for the feedback. 

Let me give a quick update about releases. We've released the [iOS SDK 1.13.2](https://github.com/Netvent/storyly-mobile/blob/master/ios/RELEASENOTES.md#1132-02082021). Please check release notes and changes, you may consider using latest version to experience new features 🎉 

Have a nice day!
